avocado/utils/rpmsetup.py

```diff
@@ -0,0 +1,161 @@
+"""
+
+SUMMARY
+------------------------------------------------------
+Utility to manipulate RPM packages.
+
+Copyright: Intra2net AG
+
+
+INTERFACE
+------------------------------------------------------
+
+"""
+
+import logging
+import re
+import os
+
+from avocado.utils import process
+
+
+class RPMInstallError(Exception):
+    pass
+
+
+class RPMEraseError(Exception):
+    pass
+
+
+def rpm_is_installed(package_name):
```

Comment:
This looks very similar to `avocado.utils.software_manager.RpmBackend.check_installed()`, have you tried using it?

Reply:
Very good hint, I will do some testing and compare the two utilities in terms of what functionality I could reuse straight from the software manager. If I find anything in the `rpmsetup` that could extend the software manager utility, I will migrate it there.

Reply:
I moved all extensions to the `software_manager.RpmBackend` now.